ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.4.yaml

```diff
@@ -185,14 +185,15 @@ tests:
         ENABLE_KUBE_API=true
         MAKEFILE_TARGET=all
     workflow: baremetalds-assisted
-- as: e2e-metal-assisted-networking
+- as: e2e-metal-assisted-static-ip
   optional: true
   run_if_changed: ^(terraform_files/.*|.*/controllers/.*)$
   steps:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
         SERVICE_BASE_REF=release-ocm-2.4
+        STATIC_IPS=true
         TEST_FUNC=test_networking
```

Comment:
just take into consideration that if we'll decide that the current ff-ing of this branch is a bug and not a feature, we'll need to backport the change when we restore the branch to its previous state

Reply:
I know. I just don't want to break another 2.4 test